README.md

```diff
@@ -97,6 +97,10 @@ For documentation on how to configure code-gov-front-end, go [here](CONFIGURATIO
 ## Questions?
 
 If you have questions, please feel free to [send us an email](mailto:code@gsa.gov).
+Twitter: @CodeDotGov
+Email: code@gsa.gov
+LinkedIn: code-gov
```

Comment:
Please link out to the linkedin page. Here's the markdown you can use:
[code-gov](https://www.linkedin.com/company/code-gov/)